middleware/v2/Makefile

```diff
@@ -64,21 +64,11 @@ install:
 ifneq ($(FLASH_SIZE_SHRINK),y)
 	# copy sample_xxx
 	@cp -f sample/mipi_tx/sample_dsi $(DESTDIR)/usr/bin
-	@cp -f sample/cipher/sample_cipher $(DESTDIR)/usr/bin
-	@cp -f sample/cvg/sample_cvg $(DESTDIR)/usr/bin
 	@cp -f sample/venc/sample_venc $(DESTDIR)/usr/bin
 	@cp -f sample/venc/sample_vcodec $(DESTDIR)/usr/bin
 	@cp -f sample/vdec/sample_vdec $(DESTDIR)/usr/bin
 endif
 
-ifneq ($(FLASH_SIZE_SHRINK),y)
-	# copy venc
-	@cp -f modules/venc/vc_lib/bin/cvi_h265_enc_test $(DESTDIR)/usr/bin
```

Comment:
@carbonfix The source of these binaries does not exist in the middleware folder, it lives in https://github.com/milkv-duo/duo-buildroot-sdk-v2/blob/develop/osdrv/interdrv/v2/cvi_vc_drv/vcodec/sample/cvi_h265_enc_test.c

Same goes for the rest of these vc_lib/bin binaries.